assets/main.js

```diff
@@ -954,7 +954,8 @@ async function updateUnikernel(job, build, unikernel_name) {
 			body: JSON.stringify(
 				{
 					"job": job,
-					"build": build,
+					"latest_build": latest_build,
+					"current_build": current_build,
```

Comment:
again, the naming... so we have a "current_unikernel" and an "update_candidate" maybe?